### ISSUE TITLE
Adding new Timestamp(rowversion) fields to temporal table with existing historioc changes not working

### ISSUE THREAD

**Issue by withinoneyear** [aspnet/EntityFramework#9403]
EF core does not append "NULL" to the generated sql script to add the new Timestamp(rowversion) column.
So if the table is system-versioned, and if it already has some changes made (saved in the history table), the db migration created for this schema change will not work. 
Even "nullable" is sepefically set to true it does not create a nullable rowversion column. However, the history table set this column to Not Null by default. The primary table can work because it generates the value automatically for this column, but history table doesn't

### Steps to reproduce
* run ef core migrations to create a new table
* run ef core migrations to conver the table to temporal table
* ef database update
* add a new row to the table
* update the row so it adds rows to history table
* alter the model to add a new Timestamp(rowversion) field
* run ef core migrations to reflect the schema change
* run ef database update to apply the change

### Further technical details
EF Core version: 1.1.1
Database Provider: (Microsoft.EntityFrameworkCore.SqlServer)
Operating system:  (Visual Studio 2017)

**Comment by bricelam** [comment]
Possibly fixed with #8146

**Comment by smitpatel** [comment]
Issue:
`rowversion` is by default non-nullable in SqlServer. It will be made nullable only when you specify `NULL` in declaration. Even with the NULL, when you add column with rowversion to existing table, SQL Server will add values for existing records. Hence `nullable rowversion` column is considered *NOT* useful.

For temporal tables, the main table acts same as normal table so can have just rowversion but history table is only inserted & never updated (perhaps SQL Server does not allow updates to it) hence it cannot insert values and hence needs nullable column.

```
ALTER TABLE only allows columns to be added that can contain nulls, or have a DEFAULT definition specified, or the column being added is an identity or timestamp column, or alternatively if none of the previous conditions are satisfied the table must be empty to allow addition of this column. 
```

Fixed via #8146

**Comment by smitpatel** [comment]
We can report this to SqlServer team.
